### ISSUE TITLE
Cannot download `hayabusa-win-x64.zip` on Windows 11

### ISSUE THREAD

**Issue by fukusuket** [Yamato-Security/hayabusa#1389]
**Describe the bug**
It's not actually Hayabusa's bug ... :( 
but the browser(Edge/Chrome) is blocking the download, so the [hayabusa-2.16.0-win-x64.zip](https://github.com/Yamato-Security/hayabusa/releases/download/v2.16.0/hayabusa-2.16.0-win-x64.zip) binary is currently not downloadable on Windows 11.

**Step to Reproduce**
1. Login Windows 11
2. Open Edge
3. Click link [hayabusa-2.16.0-win-x64.zip](https://github.com/Yamato-Security/hayabusa/releases/download/v2.16.0/hayabusa-2.16.0-win-x64.zip)

**Expected behavior**
hayabusa-2.16.0-win-x64.zip is downloadable.

**Actual behavior**
hayabusa-2.16.0-win-x64.zip is not downloadable as follows.

**Screenshots**
<img width="1440" alt="スクリーンショット 2024-07-24 20 59 17" src="https://github.com/user-attachments/assets/a097c35a-0dc6-499e-a007-e8a5d8a1f9ce">

**Environment (please complete the following information):**
- OS: Windows11
- hayabusa version 2.6.0(not only this version)

**Additional context**
- I can't download it in both Chrome and Edge.
- But [hayabusa-2.16.0-all-platforms.zip](https://github.com/Yamato-Security/hayabusa/releases/download/v2.16.0/hayabusa-2.16.0-all-platforms.zip) can be downloaded.

**Comment by fukusuket** [comment]
It may be a temporary problem, so we will see how it goes for a while.

**Comment by fukusuket** [comment]
I tried again today and was able to download the file with no problems, so the issue is closed.

**Comment by fukusuket** [comment]
It is now reproducing again... :( I'll keep an eye on it for a while longer.
![defender-error](https://github.com/user-attachments/assets/3276484d-a494-455c-90c9-fed2c4f9876e)

**Comment by yapper899** [comment]
Sorry to bother you on this but Windows Defender blocking the download Hayabusa-2.16.0-win-x64.zip 
or hayabusa-2.16.0-all-platforms.zip to my Windows 10 box

Since Win Defender is triggering on a different file in the win-x64 .zip file
than in the all-platforms zip file, I wanted to verify that the below files should be 
tagged as trojan in the related .zip files before I take them out of quarantine.
I didnt see anything about these files specifically  in the documentation

win-x64.zip file:

Trojan:Win32/Casdet!rfn
hayabusa\hayabusa-2.HWXrULIn.16.0-win-x64.zip.part

all-platforms.zip file:

Trojan:PowerShell/Fleisnam.F
\hayabusa-2.16.0-all-platforms\rules\sigma\builtin\process_creation\proc_creation_win_powershell_amsi_init_failed_bypass.yml

Trojan:PowerShell/Malgent!MSR
\hayabusa\hayabusa-2.16.0-all-platforms\rules\sigma\sysmon\process_creation
\proc_creation_win_powershell_amsi_init_failed_bypass.yml

Thank you

**Comment by YamatoSecurity** [comment]
@yapper899 Thanks for letting us know. We will remove these rules from the next package until we implement encrypting of the rules in order to get around Windows Defender blocking things.

**Comment by YamatoSecurity** [comment]
@fukusuket @yapper899 I updated the rules that do not include the rules that cause false positives here: https://github.com/Yamato-Security/hayabusa/releases/tag/v2.16.1
Please check to see if you can download it without errors.

**Comment by fukusuket** [comment]
@YamatoSecurity 
I have verified that I can download all the zips and run the exe! :) Thank you  so much!

**Comment by fukusuket** [comment]
I have confirmed that  [hayabusa-2.17.0-win-x64.zip](https://github.com/Yamato-Security/hayabusa/releases/download/v2.17.0/hayabusa-2.17.0-win-x64.zip)  can be downloaded in Widows 11 :)

**Comment by YamatoSecurity** [comment]
@fukusuket Thanks for checking! I think this was fixed after ignoring the problem rules with 2.16.1. I will close this issue for now but please re-open if you get alerts again.

**Comment by fukusuket** [comment]
It seems that Windows Defender has detected it again, and we can't download it...
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/60e45a42-d7aa-41b2-9c05-4b954a064fac" />

<img width="1920" height="952" alt="Image" src="https://github.com/user-attachments/assets/ce19d5de-c353-4e2a-a031-4e3d0fe90c01" />

**Comment by YamatoSecurity** [comment]
@fukusuket I think this is expected. The normal release binaries will be detected by Defender but the live response is encoded so should not. I don't think there is anything we can do to fix it on our end...

**Comment by fukusuket** [comment]
@YamatoSecurity 
Yes, I understand! I believe this issue could be confusing for first-time users, so I left a comment as a reminder. Closing the issue!
